Source/ISHRemote/Trisoft.ISHRemote/Objects/Public/IshSession.cs

```diff
@@ -141,13 +149,20 @@ private void CreateConnection()
             //prepare connection for authentication/authorization
             var connectionParameters = new InfoShareWcfConnectionParameters
             {
-                Credential = _ishSecurePassword == null ? new NetworkCredential() : new NetworkCredential(_ishUserName, SecureStringConversions.SecureStringToString(_ishSecurePassword)),
+                Credential = _ishSecurePassword == null ? null : new NetworkCredential(_ishUserName, SecureStringConversions.SecureStringToString(_ishSecurePassword)),
```

Comment:
"? null" should be "? new NetworkCredential" ... or was that a double initialization... was this tested? Why change?